### PR TITLE
Done

### PR DESCRIPTION
**Pull request by tomer-epstein**

### CODE REVIEW

File: backend/src/extension.ts [hunk 1]
```diff
@@ -123,25 +123,29 @@ export class YeomanUIPanel {
...
+						this.theia.isInTheia().then((value) => {
+							let commandName_Close = "workbench.action.closeActiveEditor";
+							let commandName_OpenWorkspace = "vscode.openFolder";
+							let commandParam = _.get(message, "commandParams[0]");
+							vscode.window.showInformationMessage('Where would you like to open the project?', Close , OpenWorkspace)
+								.then(selection => {
+									if (selection === Close) {
+										this.executeCommand(commandName_Close, undefined);
+									}
+									if (selection === OpenWorkspace) {
```
Comment (slavik-lvovsky) [inline backend/src/extension.ts]: change to else if
selection cannot be Close and OpenWorkspace at the same time

File: backend/src/extension.ts [hunk 2]
```diff
@@ -151,6 +155,26 @@ export class YeomanUIPanel {
 		);
 	}
 
+	private executeCommand(commandName: string, commandParam: any) {
+		this.theia.isInTheia().then((value) => {
```
Comment (slavik-lvovsky) [inline backend/src/extension.ts]: add return

File: backend/src/extension.ts [hunk 3]
```diff
@@ -151,6 +155,26 @@ export class YeomanUIPanel {
...
+				commandParam = vscode.Uri.file(commandParam);
+			}
+			if (value) {
+				const commandMappings: Map<string, string> = this.theia.getCommandMappings();
+				const theiaCommand = commandMappings.get(commandName);
+				if (theiaCommand !== undefined) {
+					commandName = theiaCommand;
+				}
+			}
+			vscode.commands.executeCommand(commandName, commandParam).then(success => {
```
Comment (slavik-lvovsky) [inline backend/src/extension.ts]: add return

File: backend/src/extension.ts (hunk 2 above)
Comment (tomer-epstein) [reply backend/src/extension.ts]: done

File: backend/src/extension.ts (hunk 1 above)
Comment (tomer-epstein) [reply backend/src/extension.ts]: done

**Pull request merged**